openms-wiki/about.md

```diff
@@ -0,0 +1,12 @@
+
+# About
+
+OpenMS is an open-source C++ library for mass spectrometry (MS) data management and analyses. It offers an infrastructure for the rapid development of mass spectrometry-related software. OpenMS is free software available under the three-clause BSD license. It runs under Windows, Mac OS X and Linux.
+
+OpenMS comes with a vast variety of pre-built and ready-to-use tools for proteomics and metabolomics data analysis (TOPP tools) and powerful 2D and 3D visualization (TOPPView). It supports analyses for various quantification protocols, including label-free shotgun proteomics, SILAC, iTRAQ, SRM, and SWATH. It also provides built-in algorithms for peptide/protein identification, both de-novo and via database searching, as well as adapters to state-of-the art tools like X! Tandem, Mascot, OMSSA and others.
```

Comment:
let's replace
1. /proteomics/Proteomics.
2. /metabolomics/Metabolomics.
3. > OpenMS comes with a vast variety of pre-built and ready-to-use tools for proteomics and metabolomics data analysis (TOPP tools) and powerful 2D and 3D visualization (TOPPView).
    
    OpenMS comes with a vast variety of pre-built and ready-to-use tools for Proteomics, Metabolomics data analysis (TOPP tools), and 2D/3D visualizations (TOPPView).
4. > OMSSA and others
    
     OMSSA, and others.
5. > state-of-the art
 
     state-of-the-art

Reply:
For point 3: https://developers.google.com/style/slashes